ui/components/app/snaps/snap-authorship-expanded/snap-authorship-expanded.js

```diff
@@ -53,6 +58,23 @@ const SnapAuthorshipExpanded = ({ snapId, className, snap }) => {
   const subjectMetadata = useSelector((state) =>
     getTargetSubjectMetadata(state, snapId),
   );
+  const snapRegistryData = useSelector((state) =>
+    getSnapRegistryData(state, snapId),
+  );
+  const { website = undefined } = snapRegistryData?.metadata ?? null;
```

Comment:
```suggestion
  const { website = undefined } = snapRegistryData?.metadata ?? {};
```
I don't think you can destructure `null`?

Reply:
True, my mistake. Updated it now.